Doc/howto/pyporting.rst

```diff
@@ -31,19 +31,19 @@ are:
 
 #. Only worry about supporting Python 2.7
 #. Make sure you have good test coverage (coverage.py_ can help;
-   ``pip install coverage``)
+   ``python -m pip install coverage``)
```

Comment:
Maybe it would be good to add a note to this section that all installation commands assume the user is in a virtual environment (with appropriate link to the glossary)? We unfortunately can't assume the readers of this HOWTO are experienced developers who know not to install globally.